server/events/comment_parser.go

```diff
@@ -364,6 +372,32 @@ Flags:
 Use "atlantis [command] --help" for more information about a command.` +
 	"\n```"
 
+// HelpCommentWithoutApply is the comment we add to the pull request when someone runs
+// `atlantis help` and apply is disabled globally.
+var HelpCommentWithoutApply = "```cmake\n" +
+	`atlantis
+Terraform Pull Request Automation
+
+Usage:
+  atlantis <command> [options] -- [terraform options]
+
+Examples:
+  # run plan in the root directory passing the -target flag to terraform
+  atlantis plan -d . -- -target=resource
+
+Commands:
+  plan     Runs 'terraform plan' for the changes in this pull request.
+           To plan a specific project, use the -d, -w and -p flags.
+  unlock   Removes all atlantis locks and discards all plans for this PR.
```

Comment:
Can you make the help string a go template that conditionally includes the apply output so we're not duplicating this string.